annexremote/annexremote.py

```diff
@@ -545,10 +549,16 @@ class Protocol(object):
 
     def __init__(self, remote):
         self.remote = remote
-        self.version = "VERSION 1"
         self.exporting = False
         self.extensions = list()
 
+    @property
+    def version(self):
+        if self.remote.exportsupported() and not self.remote.force_version1:
+            return "VERSION 2"
```

Comment:
wouldn't sending `VERSION 2` depend on git-annex version?  since if it is annex older than the one added support for `VERSION 2`, it would puke (or not)?

Reply:
That's true. Now that I'm looking into it again I we should just scrap the last two commits. `VERSION 2` basically only exists so a special remote can ensure that git-annex is new enough to not have the bug. But 78c280d1de3d1eb5c4cf6f789141d047d88ae718 already handles the bug,